README.md

```diff
@@ -27,7 +27,21 @@
 Backend code for CS320 Peer Recognition Software Prototype (Spring 2021) using the [Nest](https://github.com/nestjs/nest) framework.
 
 ## Installation
+From the root directory of the project
 
+### Docker setup
+#### stop current database
+```bash
+$ docker-compose down; docker rm -fr (docker ps | grep postgres | cut -c1-12);\
```

Comment:
removing the container and volume should only be done in the event that your backend can't connect - really should be a one time thing as part of setting up the dev environment - we want the volume to persist data and the container to still be set up when not in use. docker-compose down stops and shuts down the database without actually removing any data or destroying the container and volume so that part is correct. You may want to split this section into "stop the database" and "reset the database" making sure to make a note in the readme that resetting the database by removing the container and volume will completely get rid of all the data stored within them.